python_cross_site_scripting/QGB_QPSU/HTML.py

```
@@ -0,0 +1,2126 @@
+
+<div id="fullscreen" class="html5-fullscreen-api">
+	<img src="img_url">
+	<span class="fs-button"></span>
+</div>
+
+
+</body>
+</html>
+
+'''.replace('img_url',u) # %py.locals()
+	response.headers['Content-Type']='text/html;charset=utf-8';
```

Ruff (0.13.3)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)